### ISSUE TITLE
Allow Instructors to indicate accommodations they need to teach effectively

### ISSUE THREAD

**Issue by elichad** [carpentries/amy#2436]
Suggested by @bkmgit in #2435.

An Instructor can currently provide information about accommodations in the notes when expressing interest in teaching a workshop, but we don't explicitly ask for this anywhere. We could make this part of the instructor's AMY profile to avoid them having to provide the same information repeatedly.

**Comment by elichad** [comment]
This needs some discussion about what exact information we should ask for and store, how we handle temporary vs permanent accommodations, and what control instructors should have over sharing this information with workshop hosts.

As a simple improvement, we can update the text on the sign-up form to explicitly ask the instructor to include any accommodations they need in the notes.

**Comment by maneesha** [comment]
Marking this high priority for the small change (updating text on sign-up form).  Will consider other improvements later.
